src/audio/codec_adapter/codec/passthrough.c

```diff
@@ -81,3 +86,14 @@ int passthrough_codec_free(struct comp_dev *dev)
 
 	return 0;
 }
+
+static struct codec_interface passthrough_interface = {
+	.init  = passthrough_codec_init,
+	.prepare = passthrough_codec_prepare,
+	.process = passthrough_codec_process,
+	.apply_config = passthrough_codec_apply_config,
+	.reset = passthrough_codec_reset,
+	.free = passthrough_codec_free
+};
+
+DECLARE_CODEC_ADAPTER(passthrough_interface, passthrough_uuid, passthrough_tr)
```

Comment:
You will need a semicolon here, otherwise this won't compile. I am also looking at this. Let me know if you want me to take this over and send the next PR. 

I have tested cadence codec adapter with the previous version of your PR and it works.

Reply:
Ah that is what is missing, thanks i was struggling with that. I will update this once we figure out the corruption issues in the adapter. I don't want to land anything until those are fixed.

Reply:
@cujomalainey btw, how do you generate the UUID?

Reply:
@dbaluta I think the simple "uuidgen" tool is used, should work well enough.

Reply:
i mean, these are not cryptographically secure, you could use a webtool if desired